core/tests/unit/test_client.py

```diff
@@ -132,17 +132,17 @@ def test__http_property_new(self):
         client = self._make_one(credentials=credentials)
         self.assertIsNone(client._http_internal)
 
-        patch = mock.patch('google_auth_httplib2.AuthorizedHttp',
-                           return_value=mock.sentinel.http)
-        with patch as mocked:
+        authorized_session_patch = mock.patch(
+            'google.auth.transport.requests.AuthorizedSession',
+            return_value=mock.sentinel.http)
+        with authorized_session_patch as AuthorizedSession:
             self.assertIs(client._http, mock.sentinel.http)
             # Check the mock.
-            mocked.assert_called_once_with(credentials)
-            self.assertEqual(mocked.call_count, 1)
+            AuthorizedSession.assert_called_once_with(credentials)
             # Make sure the cached value is used on subsequent access.
             self.assertIs(client._http_internal, mock.sentinel.http)
             self.assertIs(client._http, mock.sentinel.http)
-            self.assertEqual(mocked.call_count, 1)
+            self.assertEqual(AuthorizedSession.call_count, 1)
```

Comment:
Because more stuff is done after that assert, this again asserts that it's only been called once.